config/sync/user.role.customer.yml

```diff
@@ -18,5 +21,11 @@ permissions:
   - 'create page content'
   - 'create remote_video media'
   - 'create video media'
+  - 'edit any article content'
+  - 'edit any event content'
+  - 'edit any page content'
```

Comment:
I think customer role should not have this permission.

Reply:
@tannguyen04 they need this or they can't edit any pages. Workbench Access controls what they can edit.